src/main/controllers/AcasCertNumController.ts

```diff
@@ -64,13 +64,14 @@ export default class AcasCertNumController {
   }
 
   public post = (req: AppRequest, res: Response): void => {
+    setUserCaseForRespondent(req, this.form);
     const { saveForLater } = req.body;
 
     if (saveForLater) {
-      handleSaveAsDraft(res);
+      handleSessionErrors(req, res, this.form, PageUrls.CLAIM_SAVED);
+      handleUpdateDraftCase(req, this.logger);
```

Comment:
This line is the same in both branches. Is it worth moving this outside of the if statement entirely?

Reply:
Moved outside if statement